extensions/samples/assets/shared/scripts/provisionComposer.js

```diff
@@ -465,7 +465,7 @@ const create = async (
     message: `> Create App Id Success! ID: ${appId}`,
   });
 
-  const resourceGroupName = `${name}-${environment}`;
+  const resourceGroupName = (name = `${name}-${environment}`);
```

Comment:
This re-assignment could lead to bugs because it is easy to miss and it changes a parameter mid-function.
Consider passing resourceGroupName rather than name to getDeploymentTemplateParam and getQnaTemplateParam instead.

Reply:
Good point! How about now?